extension/lib/main.js

```diff
@@ -113,14 +110,65 @@ exports.main = function (options, callbacks) {
         widget.tooltip = config.extension.widget_tooltips[data];
         break;
     }
+  });*/
+
+  var { Frame } = require("sdk/ui/frame");
+  var { ToggleButton } = require('sdk/ui/button/toggle');
+  var { Toolbar } = require("sdk/ui/toolbar");
+
+  var frame = new Frame({
+    url: "./frame/frame.html"
   });
 
+  var menuButton = ToggleButton({
+    id: "menu-button",
+    label: "Menu",
+    icon: {
+      "32" : "./addon32.png",
+      "64" : "./addon64.png"
+    },
+    onChange: handleMenuButtonChange
+  });
+
+  var logButton = ToggleButton({
+    id: "log-button",
+    label: "Logging",
+    icon: {
+      "32" : "./addon32.png",
+      "64" : "./addon64.png"
```

Comment:
Well, it should be definitely visible to the user if logging is on or off. Can we use text for the buttons too, or do only icons work? As long as we don't have icons I would propose to use text buttons.